addons/account/static/src/js/reconciliation/reconciliation_model.js

```diff
@@ -1227,6 +1228,17 @@ var StatementModel = BasicModel.extend({
         if (prop.reconcileModelId) result.reconcile_model_id = prop.reconcileModelId
         return result;
     },
+
+    /**
+     * Used as a hook to handle return values of the validate's line process.
+     * @private
+     * @param {object} params
```

Comment:
I would name it `data` instead of `params`, because those are data received from the server.
`params` makes more sense when you have several unrelated data that are passed to a func, which is not really the case here.